File: data/java/MAXIMUM_EQULIBRIUM_SUM_ARRAY_TRANSLATED.py
====================
def f_gold ( arr , n ) :
    res = np.inf
    for i in range ( n ) :
        prefix_sum = arr [ i ]
        for j in range ( i ) :
            prefix_sum += arr [ j ]
        suffix_sum = arr [ i ]
        for j in range ( n - 1 , i , - 1 ) :
            suffix_sum += arr [ j ]
        if prefix_sum == suffix_sum :
            res = max ( res , prefix_sum )
    return res

File: data/python/MAXIMUM_EQULIBRIUM_SUM_ARRAY_TRANSLATED.java
====================
static int fGold ( int [ ] arr , int n ) {
  int res = - Integer . MAX_VALUE - 1 ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    int prefixSum = arr [ i ] ;
    for ( int j = 0 ;
    j < i ;
    j ++ ) {
      prefixSum += arr [ j ] ;
    }
    int suffixSum = arr [ i ] ;
    int j = n - 1 ;
    while ( ( j > i ) && ( j < n ) ) {
      suffixSum += arr [ j ] ;
      j -- ;
    }
    if ( ( prefixSum == suffixSum ) || ( suffixSum == 0 ) ) {
      res = Math . max ( res , prefixSum ) ;
    }
  }
  return res ;
}

